src/logics/LALogic.h

```diff
@@ -46,6 +46,7 @@ class LALogic: public Logic
     static const char*  s_sort_num;
 
     Map<PTRef,bool,PTRefHash> la_split_inequalities;
+    Map<PTRef,bool,PTRefHash> constIteTerms;
```

Comment:
I had a vague idea that storing this information would allow some optimisation, but nothing of the sort is implemented.  Should be removed?

Reply:
Yes, it seems you are not using it now, so I would remove it, unless you have an idea how to use this information.